Improve text

BinaryMoon opened the pull request:
Fix spelling and grammar
Remove references to Help Scout
Standardise spellings (for example E-Mail, E-mail and Email).
Remove some white space (automatic editor setting on save)

BinaryMoon commented:
> Next time make sure to pull from master first

Will do.

> Btn-links on user Profile page - leave as is (Reset password).

Which file do you mean? I checked and views/users/profile.blade and all the html looks the same.

> Email in the app is spelled as "Email", so no need to change to "E-Mail".

It's mostly spelt Email, but in some cases it is spelt E-Mail or E-mail. See the diff here for example:
https://github.com/freescout-helpdesk/freescout/pull/156/files#diff-731fd6d11c0d36b0d7d6853b53a2b716
or here:
https://github.com/freescout-helpdesk/freescout/pull/156/files#diff-79246e6ca638fe8b51af4c413e29323cL46

freescout-helpdesk commented:
> Which file do you mean? I checked and views/users/profile.blade and all the html looks the same.

https://github.com/freescout-helpdesk/freescout/pull/156/commits/507aaeb333c455588aebeecde87448ffa94c0e6b#diff-818837c82558679abe0e8880630841f0R181

BinaryMoon commented:
Thanks - I checked that and the html looks the same. I did capitalise the words, and I removed the s from the end of Resettings, since that's wrong. I guess the capitalisation could be overkill.

I have reverted that change (apart from the resettings, which is still wrong).
